drivers/lcd/lcd_framebuffer.c

```diff
@@ -220,34 +220,40 @@ static int lcdfb_updateearea(FAR struct fb_vtable_s *vtable,
       run += (startx * pinfo->bpp + 7) >> 3;
     }
 
-  /* Update the whole screen? */
-
-  if (startx == 0 && endx == priv->xres - 1 &&
-      starty == 0 && endy == priv->yres - 1)
+  if (pinfo->putarea != NULL)
     {
-      /* Yes, LCD driver support putarea callback? */
-
-      if (pinfo->putarea != NULL)
+      ret = pinfo->putarea(pinfo->dev, starty, endy, startx, endx, run);
```

Comment:
I think the memory layout is different from the frame buffer and what's lcd driver want.

Reply:
Ok, reasonable. could you add some comment in the related place to notify the driver writer?

Reply:
OK it is ready.